src/utils/api-utils.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const allarticlesURL = 'https://nc-news-api-v5m5.onrender.com/api/articles'
+export const getAllArticles = () => {
+    return axios.get(allarticlesURL)
+    .then((response) =>  response.data.articles)
+    .catch((err) => console.log(err))
+
+}
+
```

Comment:
Good job extracting this, you can take it one step further and create a baseURL instance, there's an example in the notes :)